app/controllers/inventories_controller.rb

```diff
@@ -0,0 +1,89 @@
+class InventoriesController < ApplicationController
+  before_action :set_inventory, only: %i[show edit update destroy]
+  before_action :authenticate_user!
+
+  # GET /inventories or /inventories.json
+  def index
+    # @inventories = Inventory.all
+
+    return unless current_user
+
+    return unless current_user.present?
+
+    # @inventories = Inventory.includes([:user]).where(user: current_user).order(created_at: :desc)
```

Comment:
- [ ] let's remove commented code as it makes code hard to read.

Reply:
Following commit for the solution:

https://github.com/Linushaddai99/Group-Recipe-app/pull/33/commits/cb40a1da187ce86386e47dd0e40c0cf3546eeed9